packages/frontend/src/components/NavBar/index.tsx

```diff
@@ -66,12 +66,15 @@ const NavBar = () => {
         },
     });
 
+    const homeUrl = selectedProjectUuid
+        ? `/projects/${selectedProjectUuid}/home`
+        : '/createProject';
```

Comment:
I would prefer If we had a new route and redirect to it if the user has access to 0 projects.

Eg:
```
<Route path="/no-access">
    <NavBar />
    <TrackPage
        name={
            PageName.NO_ACCESS
        }
    >
        <ForbiddenPanel />
    </TrackPage>
</Route>
```

In `Projects.tsx` have the redirect logic:

```
if(availableProjectUuids.length === 0) {
   //redirect to /no-access
}

```

This way we don't need to handle the 403 error in each page/request.

Reply:
This page is handling the first redirection, that happens when the user first login / creates an account. 

I think it looks nicer the /createProject than just the forbidden page . 

also this `if(availableProjectUuids.length === 0) {` would happen after the `if(error)` which shows the `Unexpected error` message, so it should happen at some point before that (or inside the error if checking if statuscode is 403)

Reply:
> This page is handling the first redirection, that happens when the user first login / creates an account.
> I think it looks nicer the /createProject than just the forbidden page .

But they can't create a project until we implement project permissions. And once we have it we will have the CTA 'create project' in the error page. 

So we still need a condition here to see if the user has permission to create projects, if not redirect to 'no-access' route

> also this if(availableProjectUuids.length === 0) { would happen after the if(error) which shows the Unexpected error message, so it should happen at some point before that (or inside the error if checking if statuscode is 403)

The get projects endpoint shouldn't return an error just an empty list.